src/views/DraftCount.vue

```diff
@@ -0,0 +1,222 @@
+        <ion-title>{{ translate("Draft count")}}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content>
+      <div class="header">
+        <div class="search">
+          <ion-item lines="none" class="ion-padding">
+            <ion-label slot="start">
+              CountName
+              <p>CountId</p>
+            </ion-label>
+            <ion-button slot="end" fill="outline" color="medium">{{ translate("Rename") }}</ion-button>
+          </ion-item>
+        </div>
+        <div class="filters">
+          <ion-list class="ion-padding">
+            <ion-item>
+              <ion-icon slot="start" :icon="cloudUploadOutline"/>
+              <ion-label>{{ translate("Import CSV") }}</ion-label>
+              <input id="inputFile" class="ion-hide"/>
+              <label for="inputFile" @click="openDraftImportCsvModal">{{ translate("Upload") }}</label>
+            </ion-item> 
+            <ion-item>
+              <ion-icon slot="start" :icon="calendarNumberOutline" />
+              <ion-label>{{ translate("Due date") }}</ion-label>  
+              <ion-button slot="end" size="small" class="date-time-button" style="solid" >3rd March 2024</ion-button>
+              <ion-modal class="date-time-modal">
+                <ion-content force-overscroll="false">
+                  <ion-datetime    
+                    id="schedule-datetime"        
+                    show-default-buttons 
+                    hour-cycle="h23"
+                    presentation="date-time"
+                    value="3rd March 2024"
+                  />
+                </ion-content>
+              </ion-modal>          
+            </ion-item>
+            <ion-item>
+              <ion-icon slot="start" :icon="businessOutline"/>
+              <ion-label>{{ translate("Facility") }}</ion-label>  
+              <ion-button fill="outline" @click="openDraftSearchFacilityModal">
+                <ion-icon slot="start" :icon="addCircleOutline"/>
+                {{ translate("Assign") }}
+              </ion-button>
+            </ion-item>
+          </ion-list>
+        </div>
+      </div>
+
+      <div class="list-item product border">
+        <ion-item>
+          <ion-icon slot="start" :icon="listOutline"/>
+          <ion-input
+            :label="translate('Add product')"
+            label-placement="floating"
+            :clear-input="true"
+            value="WS-90-BL"
+          >
+          </ion-input>
+        </ion-item>
+        <ion-item lines="none">
+          <ion-thumbnail slot="start">
+            <DxpShopifyImg/>
+          </ion-thumbnail>
+          <ion-label>
+            <p class="overline">{{ translate("Search result") }}</p>
+            Internal Name
+          </ion-label>
+        </ion-item>
+        <ion-button fill="clear" slot="end">
+          <ion-icon slot="icon-only" :icon="addCircleOutline"/>
+        </ion-button>
+      </div>
+
+      <main>
+        <div class="list-item border">
```

Comment:
Instead of creating a new class border for adding a separator, we can simply add hr tag.